src/core/Directus/Config/Schema/Value.php

```diff
@@ -50,19 +48,20 @@ public function value($context)
         $value = $context[$this->key()];
 
         switch ($this->_type) {
-        case Types::INTEGER:
-            return intval($value);
-        case Types::BOOLEAN:
-            $value = strtolower($value);
-            return $value === "true" || $value ===  "1" || $value === "on" || $value === "yes" || boolval($value);
-        case Types::FLOAT:
-            return floatval($value);
-        // TODO: add support to arrays
-        case 'array':
-            return $this->_default;
-        case Types::STRING:
-        default:
-            return $value;
+            case Types::INTEGER:
+                return intval($value);
+            case Types::BOOLEAN:
+                $value = strtolower($value);
+                return $value === "true" || $value ===  "1" || $value === "on" || $value === "yes" || boolval($value);
+            case Types::FLOAT:
+                return floatval($value);
+            case Types::ARRAY:
```

Comment:
Would you mind adding some tests for this on `tests/api/Config` dir? :)

Reply:
will do tomorrow.